app/models/campaign.rb

```diff
@@ -92,6 +93,14 @@ def validate_url
     errors.add(:url, 'invalid') if inbox.inbox_type == 'Website' && !use_http_protocol
   end
 
+  def inbox_must_belong_to_account
+    return unless inbox
```

Comment:
This seems redundant. The inbox_id is a required field.

Reply:
Removing this will fail the association checks.

```
describe 'associations' do
    it { is_expected.to belong_to(:account) }
    it { is_expected.to belong_to(:inbox) }
end
```